docker/service/src/main/java/uk/nhs/adaptors/scr/clients/SpineClient.java

```diff
@@ -55,7 +55,7 @@
 
         if (statusCode != HttpStatus.ACCEPTED.value()) {
             LOGGER.error("Unexpected spine SCR POST response: {}", response);
-            throw new UnexpectedSpineResponseException("Unexpected spine send response " + statusCode);
+            throw new UnexpectedSpineResponseException("Unexpected spine sent response " + statusCode);
```

Comment:
Actually my intention here was to inform that the "send" command resulted with unexpected response. So maybe this should be "Unexpected spine 'send data' response " ?